code/game/objects/items/weapons/chewables.dm

```diff
@@ -146,6 +146,19 @@ obj/item/clothing/mask/chewable/Destroy()
 	type_butt = /obj/item/trash/spitgum
 	wrapped = TRUE
 
+/obj/item/clothing/mask/chewable/tobacco/sweet
+	name = "chewing tobacco"
+	desc = "A chewy wad of sweet tobacco. Cut in long strands and treated with syrup so it doesn't taste like an ash-tray when you stuff it in your face."
```

Comment:
```suggestion
	desc = "A chewy wad of sweet tobacco. Cut in long strands and treated with syrup so it doesn't taste like an ashtray when you stuff it in your face."
```

Reply:
gone ahead and ripped this change on all the descriptions whilst we're here changing shit in this file; bulk of that description was just copied from all the other chewing tobacco, with 1 or 2 words changed, so typo was present in all